### DISCUSSION TITLE
HTTPS for Custom Domain - Certificate Request Error - never resolves

### DISCUSSION THREAD

**Discussion by jaypalexa** [community/community#22811] [Other Feature Feedback, Questions, & Ideas]
<p>Hello – I am trying to enforce HTTPS for a custom domain (<a href="http://duperbowl.com" rel="noopener nofollow ugc">duperbowl.com</a>) for a GitHub Pages site.  I <em>think</em> I am using the same settings that are working for another custom domain (<a href="http://baseballink.com" rel="noopener nofollow ugc">baseballink.com</a>) for a GitHub pages site…so I am at a bit of a loss why the Certificate Request Error never resolves (it has been weeks/months).  Any thoughts?  TIA, JP</p>
<p>Namecheap DNS Settings:</p>
<pre><code class="lang-auto">A Record	@ 185.199.108.153
A Record	@ 185.199.109.153
A Record	@ 185.199.110.153
A Record	@ 185.199.111.153
CNAME Record  www  jaypalexa.github.io.
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/878c2008948d80c361fc84685744c043179fad6d.png" data-download-href="/uploads/short-url/jl6FRxBsOTWddRZBxaC2KcJSzUN.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4446336/181073202-89acab90-0b15-4b58-8cbd-8175c12027e2.png" alt="image" data-base62-sha1="jl6FRxBsOTWddRZBxaC2KcJSzUN" width="176" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/878c2008948d80c361fc84685744c043179fad6d_2_176x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/878c2008948d80c361fc84685744c043179fad6d_2_264x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/878c2008948d80c361fc84685744c043179fad6d_2_352x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/878c2008948d80c361fc84685744c043179fad6d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1410×4000 328 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jaypalexa** [comment]
<p>Has finally resolved with no changes on my part.  :man_shrugging: :woman_shrugging:</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/jaypalexa">@jaypalexa</a>, welcome to the GitHub Support Community!</p>
<p>Glad to hear this has been resolved for you! What likely happened was when you originally set up your custom domain a few months ago your DNS settings hadn’t fully propagated across the internet at the time GitHub attempted to generate an SSL certificate for your domain. This would have caused a failure when trying to generate your certificate.</p>
<p>GitHub will attempt to generate a certificate each time you visit the settings page of a repository that has a Pages site with a custom domain if one doesn’t exist. Now that the DNS records have had time to propagate this certificate generation request will complete successfully.</p>
<p>It’s likely that when investigating this you inadvertently triggered the solution :sweat_smile:</p>

**Comment by jaypalexa** [comment]
<p>Ah, very good information.  Will keep that in mind when setting up my next site.</p>
<p>Much obliged for your response.  Best, JP</p>

**Comment by alexdsgmoura** [comment]
no momento me ocorre o mesmo problema, muito chato isso :/
